core/templates/dev/head/editor/StateResponses.js

```diff
@@ -306,6 +306,45 @@ oppia.controller('StateResponses', [
       trainingDataService.initializeTrainingData(
         explorationId, currentStateName);
     };
+    $scope.suppressDefaultAnswerGroupWarnings = function() {
+      var interactionId = $scope.getCurrentInteractionId();
+      if (interactionId === 'MultipleChoiceInput') {
+        var choices = $scope.getAnswerChoices().length;
+        var answerGroups = responsesService.getAnswerGroups();
+        // First push all choice indices into an array
+        var choiceArray = [];
+        for (var i = 0; i < choices; i++) {
+          choiceArray.push(i);
+        }
+        /* Then cycle through answerGroup rule_specs
+        and put all of them into an array */
+        var answerArray = [];
+        for (var j = 0; j < answerGroups.length; j++) {
+          /* Make sure to check if a an answerGroup rule_spec
+          has more than one rule */
+          if (answerGroups['' + j + ''].rule_specs.length > 1) {
+            for (var k = 0;
+              k < answerGroups['' + j + ''].rule_specs.length; k++) {
+              answerArray.push(
+                answerGroups['' + j + ''].rule_specs[k].inputs.x
+              );
+            }
+          } else {
+            answerArray.push(
+              answerGroups['' + j + ''].rule_specs[0].inputs.x
+            );
+          }
+        }
+        // Then compare arrays for missing values
+        for (var l = 0; l < choiceArray.length; l++) {
+          if (answerArray.indexOf(l) < 0) {
```

Comment:
Also, could use Array.prototype.every() for this:

    return choiceArray.every(function(choiceIndex) {
      return answerArray.indexOf(choiceIndex) >= 0;
    });